libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragmentAbstract.java

```diff
@@ -42,6 +42,8 @@
     public abstract void removeMedia(String mediaId);
     public abstract boolean showSavingProgressDialogIfNeeded();
     public abstract boolean hideSavingProgressDialog();
+    // Called from block editor when a media selection is cancelled
```

Comment:
I think this comment might be a bit misleading -> it's actually called from the EditPostActivity when the activity wants to inform the block editor fragment that media selection was cancelled. Wdyt?

Reply:
You're right! I've updated the comment.